packages/design-system/components/Markdown/index.tsx

```
@@ -0,0 +1,47 @@
+import './highlight'
+import StyledMarkdown from './styled'
+import { MarkdownProps } from './types'
+
+const components = {
+  img: ({ ...props }) => <LazyLoadImage alt={props.alt} ratio="16/9" {...props} />,
+  a: ({ ...props }) => {
+    const isHttp = props.href.startsWith('http')
+
+    if (!isHttp && props.target) {
+      // eslint-disable-next-line no-param-reassign
+      delete props.target
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)